fv3gfs/physics/stencils/update_dwind_phys.py

```diff
@@ -329,7 +329,7 @@ def make_storage_from_shape(shape):
     )
 
 
-def read_index_var(index_var, savepoint):
+def read_index_var(index_var, savepoint, serializer):
```

Comment:
I'm a little worried to see serialbox code inside the package namespace `fv3gfs/physics`, since it is not a runtime dependency of fv3core. Should this be in the tests code?

Reply:
Yes, this is also an intermediate stage file. I have not gotten around to include `update_dwind_phys` in `Physics`. Will make sure that there's no serialbox statement in stencils folder.